front/app/containers/AggDropDown/HistoPanel.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import moment from 'moment';
+import { head, last, propOr, defaultTo } from 'ramda';
+import { orderBy, debounce } from 'lodash';
```

Comment:
had trouble using rxjs an ramda for the ordering and debouncing functionality, largely because of typescript complaints. open to recommendations here.